scripts/bootloader/provision.py

```diff
@@ -13,13 +13,15 @@
 from hashlib import sha256
 
 
-def generate_provision_hex_file(s0_address, s1_address, hashes, provision_address, output):
+def generate_provision_hex_file(s0_address, s1_address, hashes, provision_address, output, num_counter_slots):
```

Comment:
In case some freak is calling this directly, if we set the default value `num_counter_slots=0` old code would still be compatible with this signature.

Reply:
Will do